### PR TITLE
add Malta Blockchain Regulation Proposal to News #86

### PR DESCRIPTION
**Pull request by bling5630**
# New content for Aragon Monthly

### Closing of a referenced Issue
Closes #[86](https://github.com/aragon/aragon-monthly/issues/86)

### Content category
_News_ 

#### Content type:
_Article_ 

### Title
Malta Blockchain Regulation Proposal

### Description
This article about Malta Blockchain Regulation Proposal

### CODE REVIEW

File: docs/news/daos.md [hunk 1]
```diff
@@ -16,3 +16,10 @@ In my last article, I’ve discussed some of the most common types of blockchain
...
 [Read More](https://medium.com/daostack/decentralized-governance-first-principles-1fc6eaa492ed)| [Read More](https://blog.goodaudience.com/2018-the-dao-returns-5868a473afb0) |  |
+| [**Malta Blockchain Regulation Proposal: Legal Personality for DAOs and Smart Contracts**](https://chainstrategies.com/2018/03/11/malta-blockchain-regulation-proposal-legal-personality-for-daos-and-smart-contracts/) |
```
Comment (Smokyish) [inline docs/news/daos.md]: There should be a empty line here.

Extra ` |` in the beginning of the line.

File: docs/news/daos.md [hunk 2]
```diff
@@ -16,3 +16,10 @@ In my last article, I’ve discussed some of the most common types of blockchain
...
+|:-----------:|
+[_DAO_](tag.md) |
```
Comment (Smokyish) [inline docs/news/daos.md]: Should be `DAOs` and `daos.md`.

File: docs/news/daos.md [hunk 3]
```diff
@@ -16,3 +16,10 @@ In my last article, I’ve discussed some of the most common types of blockchain
...
+![Alt tcsdsdsext](https://chainstrategies.files.wordpress.com/2018/03/20180311-chain-2130988.jpg)  |
```
Comment (Smokyish) [inline docs/news/daos.md]: Weird alt text and doesn't follow the previous format of adding an image, but works.

**Comment by bling5630** [comment]
hello @Smokyish 

thank you for your help, I will review all of your comments,

tamas

**Comment by Smokyish** [comment]
@bling5630 It looks like the commented changes were never added to this PR?

**Comment by Smokyish** [comment]
Closing this as it doesn't fulfill the requirements of adding news to the proper file in a proper format.

**Pull request closed without merging**